ECE_VERSION

```diff
@@ -0,0 +1 @@
+2.10.0
```

Comment:
Oh no! Another `VERSION` file!
/s

I do think this is better than baking it into the Makefile though 👍

Reply:
so actually, one version is in the Makefile, the other is in a file, it make make sense to put both into separate files where we end up having `VERSION` and `VERSION_ECE`. Right now the version declaration is in the main Makefile which isn't great, but not terrible either.

Reply:
While I dislike file version things (mostly because Cloud) as long as it's only in one place it's not _too_ bad, even if that place is a `VERSION` file. The versions need to go somewhere 🤷‍♂️ 
To be clear, I'm _not_ advocating to move the other one if we don't need to as part of this 🙂